scrapers.js

```diff
@@ -40,6 +40,27 @@ const scrapers = [
       return counties;
     }
   },
+  {
+    state: 'IL',
+    country: 'USA',
+    url: 'http://www.dph.illinois.gov/sites/default/files/COVID19/COVID19CountyResults.json',
```

Comment:
This makes your scraper the primary source for information on IL state

```suggestion
    url: 'http://www.dph.illinois.gov/sites/default/files/COVID19/COVID19CountyResults.json',
    priority: 1,
```